### PR TITLE
[docs/linux] Fix dependency list

### PR DESCRIPTION
**Pull request by hudokkow**
See title.

Grabbed from https://launchpad.net/~team-xbmc/+archive/ubuntu/xbmc-nightly/+sourcefiles/kodi/2:18.0+git20180730.0202-743da9e-0bionic/kodi_18.0+git20180730.0202-743da9e-0bionic.dsc

### CODE REVIEW

File: docs/README.Linux.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ git clone https://github.com/xbmc/xbmc kodi
...
... | libcurl4-gnutls-dev | libcurl-dev, libcwiid-dev, libdrm-dev, libdbus-1-dev, libegl1-mesa-dev, libenca-dev, libflac-dev, libfontconfig-dev, libfmt3-dev | libfmt-dev, libfreetype6-dev, libfribidi-dev, libfstrcmp-dev, libgbm-dev, libgcrypt-dev, libgif-dev (>= 5.0.5), libgl1-mesa-dev | libgl-dev, libgles2-mesa-dev, libglew-dev, libglu1-mesa-dev | libglu-dev, libgnutls-dev | libgnutls28-dev, libgpg-error-dev, libinput-dev, libiso9660-dev, libjpeg-dev, liblcms2-dev, liblirc-dev | liblircclient-dev, libltdl-dev, liblzo2-dev, libmicrohttpd-dev, libmysqlclient-dev, libnfs-dev, libogg-dev, libomxil-bellagio-dev [armel], libpcre3-dev, libplist-dev, libpng12-dev | libpng-dev, libpulse-dev, libshairplay-dev, libsmbclient-dev, libsqlite3-dev, libssh-dev, libssl-dev, libtag1-dev (>= 1.8) | libtag1x8, libtiff5-dev | libtiff-dev | libtiff4-dev, libtinyxml-dev, libtool, libudev-dev, libva-dev, libvdpau-dev, libvorbis-dev, libxkbcommon-dev, libxmu-dev, libxrandr-dev, libxslt1-dev | libxslt-dev, libxt-dev, waylandpp-dev, wayland-protocols, lsb-release, nasm [!amd64], python-dev, python-pil | python-imaging, python-support | python-minimal, rapidjson-dev, swig, unzip, uuid-dev, yasm, zip, zlib1g-dev
```
Comment (Rechi) [inline docs/README.Linux.md]: - debhelper isn't needed
- libssh support was moved to a binary addon
- IMO the dependencies for the different windowing systems (X11, Wayland and GBM) should in separate lists
- maybe also distinguish between required and optional dependencies

Comment (hudokkow) [reply docs/README.Linux.md]: Agreed. However, I won't have time to do that - and test - in the next couple of days and starting Wednesday I'm off the grid until Sept 3. With very few exceptions, my wife really means off the grid.

If someone beats me to it, please close this.

Comment (Rechi) [reply docs/README.Linux.md]: Would be great if you could just fix the wrong waylandpp-dev and wayland-protocols alternatives and leave everything else unchanged.

Comment (hudokkow) [reply docs/README.Linux.md]: :+1:
I'll find the time tonight or tomorrow.

**Comment by MartijnKaijser** [comment]
@hudokkow would be nice to have this updated :)

**Comment by wsnipex** [comment]
this dependency list is ubuntu specific. The general linux readme should probably list upstream library names instead

**Comment by phunkyfish** [comment]
@hudokkow do you want to progress this or close it, or backburner?

**Comment by phunkyfish** [comment]
As there has been no comment on this PR it will be closed in 2 weeks time if no comment stating to do otherwise is added.

**Comment by phunkyfish** [comment]
Going to close this, can be reopened at a later date if required.

**Pull request closed without merging**